monkey/infection_monkey/post_breach/actions/modify_shell_startup_files.py

```diff
@@ -43,3 +52,14 @@ def __init__(self, linux_cmds, windows_cmds):
         super(ModifyShellStartupFile, self).__init__(name=POST_BREACH_SHELL_STARTUP_FILE_MODIFICATION,
                                                      linux_cmd=linux_cmds,
                                                      windows_cmd=windows_cmds)
+
+    def run(self):
+        if self.command:
+            try:
+                output = subprocess.check_output(self.command, stderr=subprocess.STDOUT, shell=True).decode()
+                if not output:
+                    output = EXECUTION_WITHOUT_OUTPUT
```

Comment:
This could probably be done on the island for all PBA's. You might end up repeating this pattern of "if not output: output = EXECUTION_WITHOUT_OUTPUT" a lot.

Reply:
I modified the logic of this slightly; can you have a look?